library/core/src/test/java/com/google/android/exoplayer2/FormatTest.kt

```diff
@@ -0,0 +1,77 @@
+package com.google.android.exoplayer2
+
+import android.os.Parcel
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.android.exoplayer2.drm.DrmInitData
+import com.google.android.exoplayer2.drm.DrmInitData.SchemeData
+import com.google.android.exoplayer2.metadata.Metadata
+import com.google.android.exoplayer2.metadata.id3.TextInformationFrame
+import com.google.android.exoplayer2.testutil.TestUtil
+import com.google.android.exoplayer2.util.MimeTypes
+import com.google.android.exoplayer2.video.ColorInfo
+import com.google.common.truth.Truth
+import org.junit.Test
+import org.junit.runner.RunWith
+
+/** Unit test for [Format].  */
+@RunWith(AndroidJUnit4::class)
+class FormatTest {
+    companion object {
+        private val initData: List<ByteArray> = listOf(
+                byteArrayOf(1, 2, 3),
+                byteArrayOf(4, 5, 6)
+        )
```

Comment:
Notice no guava dependency anymore, since collections in Kotlin are immutable by default, unless explicitly stated otherwise (in this case it would mean writing `mutableListOf`).

Reply:
Also, interestingly no notice the type-inference for `byteArrayOf` when passing `Int` values, another benefit of using Kotlin over Java.